### PR TITLE
Mask passwords with spaces

### LINKED ISSUE #413: util.MaskPassword doesn't handle passwords with spaces
Hey,

I was looking at my logs from promscale and was pleased to see the PG database password was masked, but then surprised to see only the first word was masked.

Looking at the regexes, they only match until a space is seen. Looks to me like they should match until "( \w+:|$)" is seen. 

Cheers,
Andrew

### PR DESCRIPTION
**Pull request by cevian**
fixes #413.

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ const (
 
 var (
 	maskPasswordRegex1 = regexp.MustCompile(`[p|P]assword=(\s*?)'([^']+?)'`)
```
Comment (Harkishen-Singh) [inline pkg/util/util.go:26]: Shouldn't this be space consumable as well?

File: pkg/util/util_test.go [hunk 2]
```diff
@@ -73,9 +73,12 @@ func TestMaskPassword(t *testing.T) {
 		"pass='foobar' host='localhost'":                                     "pass='foobar' host='localhost'",
 		"host='localhost' password='foobar'":                                 "host='localhost' password='****'",
 		"password:foobar  host: localhost":                                   "password:****  host: localhost",
-		"password:foo bar  host: localhost":                                  "password:**** bar  host: localhost",
+		"password:foo bar  host: localhost":                                  "password:****  host: localhost",
```
Comment (Harkishen-Singh) [inline pkg/util/util_test.go:76]: Can we have more cases like `Password: "foo bar"`, `password="foo bar"`, `Password="foo bar"` ?

**Review by Harkishen-Singh — APPROVED** [review]
:+1: , some cases if applicable?

File: pkg/util/util.go (hunk 1 above)
Comment (cevian) [reply pkg/util/util.go:26]: This works because the class is `NOT '` which includes a space

**Pull request merged**